### DISCUSSION TITLE
How about we add Virtualization and Tenancy to the zones?

### DISCUSSION THREAD

**Discussion by hbasria** [auroraresearchlab/netbox-dns#60] [Ideas]
![image](https://user-images.githubusercontent.com/567055/142287874-b2c3fde9-85c0-4bb9-bc34-4e5d8592c23b.png)

**Comment by peteeckel** [comment]
I saw your WIP PR already, and currently I'm not really sure what the use case is. 

Do you have an example handy what one would need that for? Possibly I'm missing something.
